docs/pages/blog/turbo-0-4-0.mdx

```diff
@@ -88,23 +88,23 @@ RUN npm i -g turbo
 
 # Prune the workspace for the `frontend` app
 FROM base as pruner
-COPY .git ./.git
+WORKDIR /app
 COPY . .
-RUN turbo prune --scope=frontend
```

Comment:
Based on the output below (using`json/` and `full/` directories) this should have the `--docker flag`